parquet-key-management/tests/kms_encryption.rs

```diff
@@ -0,0 +1,164 @@
+//! Integration tests for KMS encryption with delta-rs.
+//! This file tests the configuration and usage of TableEncryption and KmsFileFormatOptions
+//! in various encryption scenarios.
+
```

Comment:
These comments look like they belong in `delta_rs_kms.rs`, this file only contains the implementations of `KmsFileFormatOptions` etc.